site/content/resources/adopters.md

```diff
@@ -1,17 +1,21 @@
-# Contour Adopters
+---
+title: Contour Adopters
+layout: page
+---
 
 If you're using Contour and want to add your organization to this list, please
 [submit a pull request][1]!
 
-<a href="https://knative.dev" border="0" target="_blank"><img alt="knative.dev" src="site/img/adopters/knative.svg" height="50"></a>
+<a href="https://knative.dev" target="_blank"><img alt="knative.dev" src="../../img/adopters/knative.svg" height="50"></a>
```

Comment:
hm, looks like we have these files in two places now, would be good to consolidate the images into one location

see:
- https://github.com/izturn/contour/tree/doc/ecosystem/site/img/adopters
- https://github.com/izturn/contour/tree/doc/ecosystem/site/themes/contour/static/img/adopters

Reply:
i add it `../themes`, because the first place will not work, so may i delete the first one?